src/USER-CONP/fix_charge_update.cpp

```diff
@@ -235,6 +234,7 @@ void FixChargeUpdate::setup(int) {
     }
     return ordered_mat;
   };
+  vector_compute->compute_vector(); // TODO crash with "if (f_vec)"
```

Comment:
Hum? Was meinst du damit?

Reply:
Eigentlich muss das an der Stelle nur ausgeführt werden, wenn der B-Vektor in eine Datei geschrieben werden soll. Stürzt bei mir dann aber ab.

Reply:
OK, dann lassen wir das erstmal als TODO